src/basic/virt.c

```diff
@@ -20,29 +20,28 @@
 #include "string-util.h"
 #include "virt.h"
 
+static const char *const vm_table[] = {
```

Comment:
given this table is spare it really should be sized to `_VIRTUALIZATION_MAX` I think